forms/jqm/element.js

```diff
@@ -68,9 +71,13 @@ define(function (require) {
     renderLabel: function () {
       if (!this.$label) {
         this.$label = $('<label class="ui-input-text"></label>');
+      }
+
+      if (!this.$el.find(this.$label).length) {
```

Comment:
Can't we merge this with previous, so when there is no `$label`, create one append to `$el`